### PR TITLE
Home UI

### LINKED ISSUE #58: Home Screen -> UI
### Is your feature request related to a problem? Please describe.

Creating the UI of the screen and then linking the completed screen to the correct position in the widget tree of the app.

### PR DESCRIPTION
**Pull request by Hard-Coder05**
### Description

Home UI 

**Type of Change:** (Delete irrelevant options)
- Code
- User Interface

**Code/Quality Assurance Only** (Delete irrelevant options)

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

tested locally

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] I have added one/more required dependencies
- [x] I have formatted the code using dart format (dartfmt)

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by werainkhatri** [comment]
Missing UI video/screenshots! Always add these when doing a UI PR.

**Comment by Hard-Coder05** [comment]
https://user-images.githubusercontent.com/54059881/107868913-73bc0980-6eae-11eb-9c0f-df5884dcf589.mp4

**Comment by werainkhatri** [comment]
Is the softcoding of dimensions implemented?

**Comment by Hard-Coder05** [comment]
@werainkhatri Done everything. The PR is up for review. Also I have done all the editing as asked by Mohit.

https://user-images.githubusercontent.com/54059881/107910649-d8e02f80-6f80-11eb-9697-c5b6dc328aee.mp4

File: lib/screens/home/tabs/menu/menu.dart [hunk 1]
```diff
@@ -1,8 +1,295 @@
...
+}
+
+class _MenuScreenState extends State<MenuScreen> {
   @override
   Widget build(BuildContext context) {
-    return Container(color: Colors.red);
+    double width = MediaQuery.of(context).size.width;
+    double height = MediaQuery.of(context).size.height;
+    double heightFactor = height / 1000;
+    Log.i(tag: "Screen width: ", message: width.toString());
```
Comment (werainkhatri) [inline lib/screens/home/tabs/menu/menu.dart]:
```suggestion
```

**Review by werainkhatri — CHANGES_REQUESTED** [review]

**Review by werainkhatri — APPROVED** [review]

**Comment by werainkhatri** [comment]
Good work 👍🏼

File: lib/screens/home/tabs/menu/menu.dart [hunk 2]
```diff
@@ -1,8 +1,294 @@
+import 'package:ecellapp/core/res/colors.dart';
+import 'package:ecellapp/core/res/dimens.dart';
+import 'package:ecellapp/core/res/strings.dart';
+import 'package:ecellapp/core/utils/injection.dart';
+import 'package:ecellapp/core/utils/logger.dart';
```
Comment (werainkhatri) [inline lib/screens/home/tabs/menu/menu.dart]:
```suggestion
```

**Review by werainkhatri — APPROVED** [review]

**Pull request merged**